src/widgets/hits/__tests__/defaultTemplates-test.ts

```diff
@@ -28,7 +29,7 @@ describe('hits defaultTemplates', () => {
 
     expect(
       typeof defaultTemplates.item === 'function' &&
-        defaultTemplates.item(item, () => '')
+        defaultTemplates.item(item, {} as TemplateParams)
```

Comment:
This cast is necessary to make the test pass in its current state. We might want to drop it anyway later on and test default templates at the widget level instead of unit testing this implementation detail.